tools/deploy-to-gear/src/main.rs

```diff
@@ -59,6 +56,19 @@ struct VftArgs {
     /// Decimals of the token that will be set during initialization
     #[arg(long = "token-decimals", short = 'd', default_value = "18")]
     token_decimals: u8,
+
+    #[command(flatten)]
+    common: CommonArgs,
+}
+
+#[derive(Args)]
+struct CommonArgs {
+    /// ActorId that will be allowed to mint new tokens
+    #[arg(long = "minter")]
+    minter: Option<String>,
+    /// ActorId that will be allowed to burn tokens
+    #[arg(long = "burner")]
```

Comment:
```suggestion
    #[arg(long)]
```

and for previous one